CMakeLists.txt

```diff
@@ -105,7 +105,18 @@ add_library(libbloaty
     third_party/demumble/third_party/libcxxabi/cxa_demangle.cpp
     )
 
+  set_target_properties(libbloaty PROPERTIES
+    OUTPUT_NAME bloaty
+    PUBLIC_HEADER src/bloaty.h
```

Comment:
I don't want to install this, at least for now. I don't consider the C++ interfaces public or stable: https://github.com/google/bloaty/blob/master/src/bloaty.h#L15